charts/osdfir-infrastructure/README.md

```diff
@@ -0,0 +1,295 @@
+<!--- app-name: OSDFIR Infrastructure -->
+# OSDFIR Infrastructure Helm Chart
+
+OSDFIR Infrastructure helps setup Open Source
+Digital Forensics tools to Kubernetes clusters using Helm. 
+
+Currently, OSDFIR Infrastructure
+supports the deployment of the following tools:
+  * Timesketch; ref https://github.com/google/timesketch
+  * Turbinia; ref https://github.com/google/turbinia
+
+## TL;DR
+
+```console
+helm install osdfir-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure
+```
+> **Tip**: To quickly get started with a local cluster, see [minikube install docs](https://minikube.sigs.k8s.io/docs/start/).
+
+## Introduction
+
+This chart bootstraps a OSDFIR Infrastructure deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.2.0+
+- PV provisioner support in the underlying infrastructure
+
+> **Note**: Currently Turbinia only supports processing of GCP Persistent Disks and Local Evidence. See [GKE Installations](#gke-installations) for deploying to GKE.
+
+## Installing the Chart
+
+To install the chart, specify any release name of your choice. For example, if you
+want to install the chart for development, you can choose a release name of `osdfir-dev` then run:
+```console
+helm install my-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure
+```
+The command deploys OSDFIR Infrastructure on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured 
+during installation or see [Installating for Production](#installing-for-production) 
+for a recommended production installation.
+
+## Installing for Production
+
+Pull the chart locally and review the `values-production.yaml` file for a list of values that will be used for production. 
```

Comment:
Ah no so that should be local to each chart, forgot to add the osdfir-infrastructure one in, but I just added it. I think because of them being subcharts for osdfir then the value mapping wont work the same when referencing them directly